3-fleetscope/modules/env_baseline/kueue.tf.example

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  fleet_membership_regex = "projects/([^/]+)/locations/([^/]+)/memberships/([^/]+)"
+}
+
+module "kueue" {
+  for_each                 = toset(var.cluster_membership_ids)
+
+  source                   = "../install_kueue"
+  url                      = "https://github.com/kubernetes-sigs/kueue/releases/download/v0.10.1/manifests.yaml"
```

Comment:
suggestion: Might be useful to break out the version string so it can be more easily updated.